.github/workflows/check-with-pytest-linux.yaml

```diff
@@ -2,25 +2,26 @@ name: Install locally and run pytest on Linux
 
 on:
   push:
-    branches: [ "main" ]
+    branches: ["main"]
   pull_request:
-    branches: [ "main" ]
+    branches: ["main"]
 
 jobs:
   build:
 
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [ "3.7", "3.8", "3.9", "3.10", "3.11", "3.12" ]
+        python-version: ["3.9", "3.10", "3.11", "3.12", "3.13"]
 
     steps:
       #      - name: Setup GStreamer
       #        id: setup_gstreamer
       #        uses: blinemedical/setup-gstreamer@v1.4.0
-
-      - name: Start PulseAudio server
+      # TODO(Martin): Migrate to pipewire-pulse
+      - name: Start PulseAudio server - TODO migrate to pipewire-pulse!
```

Comment:
I suggest that you add a separate test that uses `pipewire-pulse` in the future instead of modifying this one, which uses pulseaudio.

Reply:
Makes sense.

The PA backend is in maintenance mode since start of 2024 though, so it will probably make sense to be removed eventually with enough time - https://lists.freedesktop.org/archives/pulseaudio-discuss/2024-January/032426.html